lib/dom-helpers.mjs

```diff
@@ -0,0 +1,19 @@
+/**
+ * Creates an element with the given attributes
```

Comment:
I'm not 100% sure that we need this abstraction, and it could end up trying to do too much if we extend it to do everything we might want to do with a created element as the TODOs are hinting at.

Are we sure we want to go down this route rather than just using the native APIs directly in the tests?